clang/test/CodeGenCXX/address-space-cast-coerce.cpp

```diff
@@ -46,9 +46,9 @@ int mane() {
     char1 f1{1};
     char1 f2{1};
 
-// CHECK: [[TMP:%.+]] = alloca i16
```

Comment:
The revised version of casting integers is a bit more aggressive; it's hard to make it precisely match the old code while still preserving the correct semantics.